Resources/public/css/theme/views/dashboard/mycontent.css

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockmycontentview .ez-block-option-view:before {
+    content: '\E607';
```

Comment:
btw this also means you should probably remove the stylesheet in `bundles/ezplatformui/css/theme/views/dashboard/allcontent.css`